chrome-plugin/code_later.js

```diff
@@ -74,64 +74,69 @@ else if(us.length > 4 && us[2].split(".")[1] === "spoj" && us[us.length-2] === "
     var code = us[us.length - 2];
     if(code !== "classical" && code !== "challenge" && code !== "partial" && code !== "tutorial" && code !== "riddle")
     {
-        document.getElementsByClassName("problems")[0].getElementsByTagName("tbody")[0].innerHTML = 
-        document.getElementsByClassName("problems")[0].getElementsByTagName("tbody")[0].innerHTML 
-        + "<tr class='navigation'>"
-        + "<td><a id='code_now_id_java_button'>Code in Java</a></td>"
-        + "<td><a id='code_now_id_c_button'>Code in C</a></td>"
-        + "<td><a id='code_now_id_c++_button'>Code in C++<a></a></td>"
+        //document.getElementsByClassName("problems")[0].getElementsByTagName("tbody")[0].innerHTML = 
+        //document.getElementsByClassName("problems")[0].getElementsByTagName("tbody")[0].innerHTML 
+        //+ "<tr class='navigation'>"
+        //+ "<td><a id='code_now_id_java_button'>Code in Java</a></td>"
+        //+ "<td><a id='code_now_id_c_button'>Code in C</a></td>"
+        //+ "<td><a id='code_now_id_c++_button'>Code in C++<a></a></td>"
+        //+ "</tr>"
+document.getElementsByClassName("text-center")[0].innerHTML= document.getElementsByClassName("text-center")[0].innerHTML +  "<tr class='navigation'>"
```

Comment:
Getting elements with generic classes like text-center might not be a good idea - Because, it is highly likely that new "text-center" elements might be added by the SPOJ developers. But looking at the latest SPOJ frontend- I think other ways will also involve generic classes